apps/expo/src/utils/workout-tracker-utils.ts

```diff
@@ -0,0 +1,101 @@
+import { WorkoutTemplateInfoSchema } from '^/packages/validators/src'
+import { z } from 'zod'
+
+import { TExercise } from '~/components/tracker/workout-tracker'
+
+export type TWorkoutTemplateInfo = z.infer<typeof WorkoutTemplateInfoSchema>
+
```

Comment:
🥹 all the parsing type is broken. I can't make it to show anything on the tracker

Reply:
also, I can help you write some minimum unit tests for this